cloud/amazon/ec2_placement.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: ec2_placement
+short_description: create, remove or list placement group(s) in EC2.
+description:
+    - Creates, removes and lists placement groups in EC2. This module has a dependency on python-boto.
+version_added: "2.0"
+options:
+  name:
+    description:
+      - The EC2 placement group name.
+    required: False
+    default: null
+    aliases: []
+  strategy:
+    description:
```

Comment:
Just like in boto doc: http://boto.readthedocs.org/en/latest/ref/ec2.html#boto.ec2.connection.EC2Connection.create_placement_group